README.md

```diff
@@ -221,3 +224,81 @@ Depending on the input parameters, the SWS may decide one of the following:
 - `HUMAN_ON_THE_LOOP`: "Inform supervisor, who may potentially intervene"
 - `HUMAN_IN_THE_LOOP`: "Let the worker decide"
 - `HUMAN_MANUALLY`: "Let the worker carry the box"
+
+___
+
+### Construction Scenario
+
+This example scenario is from the construction domain:
+
+- imagine a robot doing work on a construction site
+- while moving, the robot detects an unexpected obstacle in its way (e.g. some construction materials that are stored there temporarily)
+- depending on the situation, either of the following may now happen:
+  - the robot may be able to autonomously circumnavigate the obstacle and continue its work
+  - the robot may be blocked and thus unable to continue its work, so that it requires human support
+- depending on the situation, SWS should support the decision if/when a human should be called for help.
```

Comment:
I would delete this line as the same information is more or less duplicated in the next line.